brainstormforce/astra
# AST-3174 - fix: double border on below and primary footer.

### Description


- Double border issue exists for below and primary footer.
- JIRA - https://brainstormforce.atlassian.net/browse/AST-3174
- Separator is missing in between border and background color for above header
- JIRA - https://brainstormforce.atlassian.net/browse/AST-3175


### Screenshots

Double Border Issue
Bug - https://d.pr/i/XfpQfj
Fix - https://d.pr/i/hPHg5w

### Types of changes




- Earlier we had fixed this issue for Above Footer but same happens in case of Below and Primary
- https://github.com/brainstormforce/astra/pull/5025

### How has this been tested?

1. Add widgets / elements in above footer.
2. Set Top Border Width option from Design Tab.
3. Toggle Inner Container Layout option to either “Stack“ or “Inline“
4. Double border gets applied to below or primary footer in preview, which should not be preset.

### Checklist:
